lib/class-wp-theme-json-gutenberg.php

```diff
@@ -3818,6 +3818,10 @@ private static function convert_variables_to_value( $styles, $values ) {
 				continue;
 			}
 
+			if ( empty( $style ) ) {
```

Comment:
TODO:

Fix this in another PR. 

Noticed when using `gutenberg_get_global_styles( array( 'typography', 'fontSize' ), array( 'block_name' => 'core/search', 'transforms' => array( 'resolve-variables' ) ) );` below.

```
Deprecated: strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated in /Users/ramon/projects/gutenberg/lib/class-wp-theme-json-gutenberg.php on line 3825

Deprecated: preg_match_all(): Passing null to parameter #2 ($subject) of type string is deprecated in /Users/ramon/projects/gutenberg/lib/class-wp-theme-json-gutenberg.php on line 3828
```

Looks like it'll occur if there's a style with the value of `null`, e.g., 

```
array(1) {
  ["blockGap"]=>
  NULL
}
```

Reply:
https://github.com/WordPress/gutenberg/pull/59258